src/js/state/getPersistable.ts

```diff
@@ -12,6 +12,7 @@ const WINDOW_PERSIST: StateKey[] = [
   "launches",
   "pluginStorage",
   "queries",
+  "draftQueries",
   "tabHistories",
```

Comment:
Why not persist the drafts too? This will help when we reload the page and see the "this draft query is gone!" error.

Reply:
As discussed, I'm down for this. We just need to make sure we also delete drafts if they are explicitly closed since I think we were depending on this to clean up for us.